examples/rust/Cargo.toml

```diff
@@ -2,4 +2,5 @@
 members = [
   "tracecon",
   "xdp",
+  "bzsstack",
```

Comment:
it's hard to pronounce it, but also why using a different name from C example? Just "profile"?